README.md

```diff
@@ -339,7 +339,8 @@ This is made possible in `Willow` through the `LogWriter` protocol.
 ```swift
 public protocol LogWriter {
     func writeMessage(_ message: String, logLevel: LogLevel)
-    func writeMessage(_ message: Message, logLevel: LogLevel)
+    func writeMessage(_ message: LogMessage, logLevel: LogLevel)
```

Comment:
Adds support for writing log messages from the LogMessage type instead of just as strings.